list-cookies/README.md

```diff
@@ -0,0 +1,9 @@
+# list-cookies
+
+## What it does
+
+This extensions list the cookies in the mozilla developers domain
```

Comment:
-> " This extensions lists the cookies in the developer.mozilla.org domain."

...or something like that. But also, see my main comment: https://github.com/mdn/webextensions-examples/pull/114#issuecomment-255877894.